conan/cli/commands/list.py

```diff
@@ -7,9 +7,9 @@
 from conan.cli.command import conan_command, OnceArgument
 from conan.cli.formatters.list import list_packages_html
 from conan.errors import ConanException
+from conans.client.graph.graph import CONTEXT_BUILD, CONTEXT_HOST
```

Comment:
Do not import in ``conan/cli`` from ``from conans``. 
I am in the process of removing this dependency. If necessary use the "Build" and "Host" string values instead.

Reply:
🤦‍♂️